### DISCUSSION TITLE
Waiting for github education benefits

### DISCUSSION THREAD

**Discussion by alexciuca** [community/community#146078] [GitHub Education]
Hello, I have been waiting a couple of days for the github education benefits to pull through since it is required to be used in class. Please can someone help me with the benefits. Thank you

**Comment by BhathiyaPrasad** [comment]
how long you waited because I also wait like 3 days to receive my  benefits

**Reply by alexciuca** [reply]
> how long you waited because I also wait like 3 days to receive my  benefits

I waited like 3-4 days, did you receive your benefits in that time span?

**Reply by BhathiyaPrasad** [reply]
yep, I received waited little more if itis not available you can contact 
https://support.github.com/

**Comment by davevad93** [comment]
Hi @alexandru356 ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @alexandru356 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
